### DISCUSSION TITLE
Consider putting workflow on Workflowhub?

### DISCUSSION THREAD

**Discussion by SarahBeecroft** [metagenome-atlas/atlas#602] [General]
Hi Silas, 

Thanks for all the active development you're doing on Metagenome-atlas. I've been helping a user group at our HPC centre install your pipeline. I wondered if you would be interested in putting your pipeline on WorkFlowHub https://workflowhub.eu to improve findability. As part of the Australian BioCommons initiative, we are trying to get all our big workflows listed on there :) 

Anyway, totally up to you, just a request. 

Thanks, 

Sarah

**Comment by SilasK** [comment]
I think that is a good idea. However how should the config files be addes specified ?

**Reply by SarahBeecroft** [reply]
Great! 

I am not 100% sure what you mean sorry, but I think you can load the whole git repo here:
https://workflowhub.eu/workflows/new#git-repo
